.lycheeignore

```diff
@@ -8,3 +8,14 @@ http://validurltoconfiglocation/
 https://mycustomdependencylocation/
 http://tools.ietf.org/html/rfc6749#section-3.2
 @parameters.json
+https://adp/
```

Comment:
bare with my issues: could we maybe order the list by type for readability? (e.g. links first e-mail after)

Reply:
no issues in housekeeping ;) fixed.